static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -273,9 +291,17 @@ export default function EndpointOverview() {
                   {t('Performance Only')}
                 </SegmentedControl.Item>
               </SegmentedControl>
+              <Button
+                style={{position: 'absolute', right: 0}}
```

Comment:
Is there a better way to do this without position absolute?

Reply:
Yup, we could use flex instead! I'm going to make that update.

Reply:
updated